### DISCUSSION TITLE
Extra lenient circles and sliders in the editor?

### DISCUSSION THREAD

**Discussion by OwenCMYK** [ppy/osu#27396] [Ideas]
Slider head accuracy was added to the game, but there were some mapping situations where it was previously useful that are somewhat difficult now. For example: BPM changes or weird rhythms would ideally be extra lenient with timing. So what if we had the ability to make a circle in the editor that was extra lenient with timing? Either by doubling the hit window, or simply making any time within the hit window score a 300 (similar to before sliderhead accuracy)

**Comment by cihe13375** [comment]
I personally support this, but https://github.com/ppy/osu/discussions/26753#discussioncomment-8270405 so idk

I recall that in old days it was rumored that lazer will introduce per-object CS, AR, etc. A potential problem is visualization though, i.e. how could the player tell if a circle has a lower OD (larger hitwindow).

**Reply by OwenCMYK** [reply]
Yeah, that's some pretty solid evidence he's not gonna add this unfortunately.

As for visualization, I actually don't think this option needs to be visualized. When a mapper places this note, they do so under the assumption that the player will click it at the expected time like any other note, but since the expected time is unintuitive, it has extra leniency. If you communicate to the player that the note is at a weird time, it may influence when they hit the note, which would ironically defeat the purpose of it in the first place. So I think it's fine if players don't even notice it, especially considering it's niche usefulness.

**Reply by bdach** [reply]
> So I think it's fine if players don't even notice it, especially considering it's niche usefulness.

and then we get issue reports of "why did i miss this slider?" and we have to ask the person what map they were playing and track down that particular slider and tell them that it's a special snowflake slider that doesn't work like most other sliders elsewhere.

i'm weakly opposed to the entire "weak object" concept because back when i played the game i didn't realise any of the "leniency", my intuitive understanding of slider was what is being called "slider head accuracy". but i suppose that is my subjective opinion. but i am _strongly_ against magic mechanics that are in no way communicable to the user unless they open editor and click on this particular slider to check if it's a special one.

**Reply by OwenCMYK** [reply]
No because you would never miss these when you would otherwise hit a normal slider. These have a strictly more lenient hit window, and I don't think players will complain that they got extra points. However, I do see your point, and I'm not opposed to the idea of adding an indicator, I just don't think it's strictly necessary for how this object is intended to be used. It works on the same principle as things like coyote time (in platformer games) or slider end leniency here in osu!, where it doesn't inherently need to be noticed because as long as it's giving the player extra leniency, they're not going to be annoyed by it.
